mne/io/edf/tests/test_edf.py

```diff
@@ -243,4 +246,20 @@ def test_write_annotations():
 
     assert_raises(RuntimeError, read_raw_edf, edf_path, preload=False)
 
+
+def test_edf_stim_channel():
+    """Test stim channel for edf file
+    """
```

Comment:
I kept the same convention as the other tests in this file.
I updated all of them.